pom.xml

```diff
@@ -44,6 +44,11 @@
         <artifactId>RosettaCore</artifactId>
         <version>3.11.8</version>
       </dependency>
+      <dependency>
+        <groupId>com.hubspot</groupId>
+        <artifactId>immutable-collection-encoding</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
This is a public github project, so it should use release versions of things in other repos.

Reply:
Actually, I think this should just be deleted now